views/layout.hbs

```diff
@@ -1,69 +1,21 @@
-    <!--[if IE 7]><link href="{{{ assetPath }}}css/govuk-template-ie7.css?0.16.0" media="screen" rel="stylesheet" type="text/css" /><![endif]-->
-    <!--[if IE 8]><link href="{{{ assetPath }}}css/govuk-template-ie8.css?0.16.0" media="screen" rel="stylesheet" type="text/css" /><![endif]-->
-
+    <link href="{{{ assetPath }}}css/govuk-template.css?0.16.0" media="screen" rel="stylesheet" type="text/css" />
     <link href="{{{ assetPath }}}css/govuk-template-print.css?0.16.0" media="print" rel="stylesheet" type="text/css" />
     <link rel="stylesheet" href="css/styles.css" />
-
-    <!--[if IE 8]>
-    <script type="text/javascript">
-      (function(){if(window.opera){return;}
-       setTimeout(function(){var a=document,g,b={families:(g=
-       ["nta"]),urls:["{{{ assetPath }}}css/fonts-ie8.css?0.16.0"]},
-       c="{{{ assetPath }}}javascripts/vendor/goog/webfont-debug.js?0.16.0",d="script",
-       e=a.createElement(d),f=a.getElementsByTagName(d)[0],h=g.length;WebFontConfig
-       ={custom:b},e.src=c,f.parentNode.insertBefore(e,f);for(;h=h-1;a.documentElement
-       .className+=' wf-'+g[h].replace(/\s/g,'').toLowerCase()+'-n4-loading');},0)
-      })()
-    </script>
-    <![endif]-->
-    <!--[if gte IE 9]><!-->
-      <link href="{{{ assetPath }}}css/fonts.css?0.16.0" media="all" rel="stylesheet" type="text/css" />
-    <!--<![endif]-->
-
-
-    <!--[if lt IE 9]>
-      <script src="{{{ assetPath }}}javascripts/ie.js?0.16.0" type="text/javascript"></script>
-    <![endif]-->
-
+    <link href="{{{ assetPath }}}css/fonts.css?0.16.0" media="all" rel="stylesheet" type="text/css" />
     <link rel="shortcut icon" href="{{{ assetPath }}}images/favicon.ico?0.16.0" type="image/x-icon" />
-
-    <!-- Size for iPad and iPad mini (high resolution) -->
-    <link rel="apple-touch-icon-precomposed" sizes="152x152" href="{{{ assetPath }}}images/apple-touch-icon-152x152.png?0.16.0">
-    <!-- Size for iPhone and iPod touch (high resolution) -->
-    <link rel="apple-touch-icon-precomposed" sizes="120x120" href="{{{ assetPath }}}images/apple-touch-icon-120x120.png?0.16.0">
-    <!-- Size for iPad 2 and iPad mini (standard resolution) -->
-    <link rel="apple-touch-icon-precomposed" sizes="76x76" href="{{{ assetPath }}}images/apple-touch-icon-76x76.png?0.16.0">
-    <!-- Default non-defined size, also used for Android 2.1+ devices -->
-    <link rel="apple-touch-icon-precomposed" href="{{{ assetPath }}}images/apple-touch-icon-60x60.png?0.16.0">
-
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <meta property="og:image" content="{{{ assetPath }}}images/opengraph-image.png?0.16.0">
-
     {{{ head }}}
   </head>
 
   <body class="{{ bodyClasses }}">
     <script type="text/javascript">document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');</script>
```

Comment:
are you doing anything with this? you'd probably be better off with `<noscript>` messaging if they don't have javascript enabled. you may also have race conditions when the `{{ bodyClasses }}` gets set